src/graphql/loginUser.ts

```diff
@@ -109,6 +121,9 @@ export async function assumeRole(
     roleId: string;
   },
 ): Promise<void> {
+  // these routes have a low rate limit
+  await sleepPromise(1000 * 12);
```

Comment:
Why not just do a retry with promise here?

Reply:
would be nicer - i can take a look to see if we have some x-retry-in header or somethin

Reply:
We have the `X-RateLimit-Reset` header: https://github.com/transcend-io/main/blob/d13f52158f2fa5377a875f41e9500dfb58a919e5/packages/backend/src/middlewares/rateLimitMiddleware.ts#L93

Reply:
epic - good callout!